app/views/shared/_api_selector.html.slim

```diff
@@ -4,7 +4,7 @@ javascript:
   document.addEventListener('DOMContentLoaded', function () {
     window.ApiSelector({
       currentApi: #{json @service},
-      apis: #{json @api_selector_services},
+      apis: #{json api_selector_services},
```

Comment:
It is not a helper yet. `helper_method :api_selector_services` in your controller

Reply:
I tested it and it worked 🤔

Reply:
This is from a `console.log(#{json api_selector_services})` in this same file, right below the `window.ApiSelector` call
![image](https://user-images.githubusercontent.com/11318903/46729183-e8e6f680-cc84-11e8-869b-786e8fa11d07.png)

Reply:
Really? Where is it defined as helper then?

Reply:
Wait OK you moved it in app/helpers/menu_helper.rb
👍